official/utils/flags/_performance.py

```diff
@@ -164,6 +169,30 @@ def _check_loss_scale(loss_scale):  # pylint: disable=unused-variable
 
       return loss_scale > 0
 
+    if fp16_implementation:
```

Comment:
which code paths does this flag effect? estimator + resnet? doesn't seem to change the keras path yet right? should we clarify?

Reply:
Only affects estimator for now. Added comment to clarify.